extra/3-magic-8-ball.js

```diff
@@ -45,8 +45,12 @@
 
 // This should log "The ball has shaken!"
 // and return the answer.
-function shakeBall() {
+function shakeBall(answer) {
   //Write your code in here
+   if ( answer === positive ) {
+      
+  console.log("The ball has shaken " + answer );
```

Comment:
Here, only "The ball has shaken!" should be in the console log. The answer should be picked at random from the possible answers.